scoper.php

```diff
@@ -15,7 +15,17 @@
 return [
     'prefix' => 'RevealPrefix' . $timestamp,
     // @see https://github.com/humbug/php-scoper/blob/master/docs/configuration.md#exposed-symbols - regex format
-    'exclude-namespaces' => ['#^PHPStan#', '#^Reveal#', '#^PhpParser#'],
+    'exclude-namespaces' => [
+        '#^PHPStan#',
+        '#^Reveal#',
+        '#^PhpParser#',
+        '#^Symplify\\\\Astral#',
+        '#^Symplify\\\\SmartFileSystem#',
+        '#^Symplify\\\\PackageBuilder#',
+    ],
+    'exclude-classes' => [
+        'Latte\Parser',
+    ],
```

Comment:
This would actually break run on Latte 2 and 3. Those classes have different API, that's why we have to scope them.

Reply:
hm, I understand... I was trying to solve this error:
```
Service of type Reveal\LattePHPStanCompiler\LatteToPhpCompiler: Service of type RevealPrefix20220711\Latte\Parser required by $latteParser in LatteToPhpCompiler::__construct() not found. Did you add it to configuration file?
```

Latte\Parser is registered as service in `packages/latte-phpstan-compiler/config/services.neon`
```neon
services:
    # latte
    - Latte\Parser
```

Maybe we could create RevealLatteParser class which will just extends Latte\Parser and register it as service?

Reply:
I see. In that case it should be prefixed in the config as well. Probably with custom callback.

---

> Maybe we could create RevealLatteParser class which will just extends Latte\Parser and register it as service?

That's also option, but we should not bend the design just to make tool happy :). There is no application reason to extend it, unless we want to modify functionality.

Reply:
Well unless somebody needs latte\parser as dependency in their services :) I hope there is no such case. 

Is there some example of such custom scoper callback?

Reply:
ok I moved it to patchers, hope it is correct O:-) Is there some simple way how to test it? I don't want to reproduce all steps from github workflows